elements/designer-mini-nav/designer-mini-nav.html

```diff
@@ -47,6 +47,46 @@
 
   Polymer({
     is: 'designer-mini-nav',
+
+    properties: {
+
+      expanded: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+      },
+
+      _collapsing: {
+        type: Boolean,
+        value: function() { return false; },
+      },
+
+    },
+
+    listeners: {
+      'designer-mini-nav-collapse': '_onCollapse',
```

Comment:
It's not drag-and-drop specific. We certainly might want the document tree to allow d-n-d reordering without collapsing the mini-nav, while palette items do collapse.

Of course, the most general thing is just to no support this kind of auto-collapsing at all, so that you have to drag out of the mini nav before it collapses, but I thought it was a nice touch since your intended target could be under the nav.